tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -470,7 +471,7 @@ public async Task HandlePlaybackRequest(string recordingId, HttpRequest incoming
             }
         }
 
-        public static async Task<RecordEntry> CreateEntryAsync(HttpRequest request)
+        public static async Task<Tuple<RecordEntry, byte[]>> CreateEntryAsync(HttpRequest request)
```

Comment:
This will use a ValueTuple rather than a Tuple to avoid allocating
```suggestion
        public static async Task<(RecordEntry, byte[])> CreateEntryAsync(HttpRequest request)
```

Reply:
committed basically exactly this